### ISSUE TITLE
init standalone projects with package.json (Like Blogs, Portfolios)

### ISSUE THREAD

**Issue by the-pesar** [vuejs/vitepress#3256]
### Is your feature request related to a problem? Please describe.

I am just learning VitePress. In the [installation](https://vitepress.dev/guide/getting-started#installation) section, I noticed that there is only existing project installation method and when I try to create a sample blog using VitePress, the `package.json` file is not generated.
I think this Developer Experience is not appropriate and confuses developers. It is too boring to add manually vitepress, package.json, tsconfig and more tools and configs for project.

### Describe the solution you'd like

We can either improve the description of [installation](https://vitepress.dev/guide/getting-started#installation) section , or support this situation in VitePress init.

Thanks.

**Comment by wChenonly** [comment]
use https://vitepress.dev/guide/getting-started#setup-wizard

**Comment by the-pesar** [comment]
it doesn't create package.json and other configs !!!
@wChenonly

**Comment by wChenonly** [comment]
> it doesn't create package.json and other configs !!! @wChenonly

It seems that there is no dependence. I try to mention pr, and I think it's better to add it.
<img width="407" alt="image" src="https://github.com/vuejs/vitepress/assets/37658262/3ceeda81-8161-4bea-8dd8-ca4c0b46e51a">

**Comment by zhangyx1998** [comment]
AFAIK,

Before you can run `npx vitepress init`, your should have either:

1. Installed vitepress **globally**. In this case it does not make sense to install a duplicate copy under your workspace folder.

2. Installed vitepress using `npm install -D vitepress` (as stated in official doc). This command will add vitepress to devDependency.

Otherwise `npx` should not even have a copy of vitepress on your computer.

**Comment by wChenonly** [comment]
> AFAIK,
> 
> Before you can run `npx vitepress init`, your should have either:
> 
> 1. Installed vitepress **globally**. In this case it does not make sense to install a duplicate copy under your workspace folder.
> 2. Installed vitepress using `npm install -D vitepress` (as stated in official doc). This command will add vitepress to devDependency.
> 
> Otherwise `npx` should not even have a copy of vitepress on your computer.

No, it doesn't have to be installed globally. People who use it for the first time just want to see a demo.

Use

1. npx vitepress init

2. npm i

3. npm run dev

Instead of going to `npm install -D vitepress`，然后在 npm run dev
